addons/l10n_it_edi_sdi_coop/models/account_edi_proxy_client.py

```diff
@@ -0,0 +1,94 @@
+from odoo import models, fields
+from .account_edi_proxy_auth import OdooEdiProxyAuth
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization
+import requests
+import uuid
+import base64
+
+
+class account_edi_proxy_client(models.Model):
+    _name = 'account_edi_proxy_client.user'
```

Comment:
I don't know... The idea was to be as generic as possible since we will want to use that later, and the model is generic (not dependent on l10n_it). Creating the new module seemed a bit premature, but I agree that this is an in-between solution and maybe not the best. Creating the new module now would avoid us to have a migration later. Do you think that's reason enough or should I change to l10n_it_edi ?

Reply:
As discussed, create a new module.